react/src/components/organisms/HeaderSlim/index.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import './style.css';
+
+const HeaderSlim = (props) => (
+  <div className="ma__header_slim">
+    <div className="ma__header_slim__banner" />
+    <header className="ma__header" id="header">
```

Comment:
i think you want to make this specific to this component, it is inheriting the top margin (gray area) - which we do not want for the slimmed header